src/CloudyCooling.hpp

```diff
@@ -81,7 +81,7 @@ AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE auto cloudy_cooling_function(Real const
 {
 	// interpolate cooling rates from Cloudy tables
 	const Real rhoH = rho * cloudy_H_mass_fraction; // mass density of H species
-	const Real nH = rhoH / quokka::hydrogen_mass_cgs;
+	const Real nH = rhoH / C::m_u;
```

Comment:
I forgot about this part of the code. When computing the hydrogen number density, we do need to use the hydrogen mass, not amu.